semantic_conventions/resource/k8s.yaml

```
@@ -10,6 +10,12 @@ groups:
         brief: >
           The name of the cluster.
         examples: ['opentelemetry-cluster']
+      - id: version
+        type: string
+        brief: >
+          The version of the cluster.
```

```suggestion
          The version of the Kubernetes implementation of the cluster.
```
or
```suggestion
          The Kubernetes version of the of the cluster.
```
How about this to make clear this is about k8s and not the version of the installed app?